src/Provider/I18nProvider.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Provider;
+
+use App\ApplicationParameters;
+use App\Service\MessageTranslator;
+use App\Service\Parameters;
+use Psr\Container\ContainerInterface;
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Yiisoft\Aliases\Aliases;
+use Yiisoft\Di\Container;
+use Yiisoft\Di\Support\ServiceProvider;
+use Yiisoft\I18n\TranslatorInterface;
+use Yiisoft\I18n\Message\PhpFile;
+use Yiisoft\I18n\Translator\Translator;
+
+final class I18nProvider extends ServiceProvider
+{
+    private string $locale;
+    private string $translatePath;
+
+    public function __construct(string $locale = 'en-US', string $translatePath = '@message')
+    {
+        $this->locale = $locale;
+        $this->translatePath = $translatePath;
+    }
+
+    /**
+     * @suppress PhanAccessMethodProtected
+     *
+     * @param Container $container
```

Comment:
i think it's not necessary `@param Container $container`

Reply:
done.